apps/admin/frontend/src/screens/print_test_deck_screen.tsx

```diff
@@ -241,6 +247,9 @@ export function PrintTestDeckScreen(): JSX.Element {
   const userRole = auth.user.role;
   const { election, electionHash } = electionDefinition;
   const [printIndex, setPrintIndex] = useState<PrintIndex>();
+  const [isSaveModalOpen, setIsSaveModalOpen] = useState(false);
+  const [precinctToSaveToPdf, setPrecinctToSaveToPdf] =
+    useState<PrecinctId>('all');
```

Comment:
We do have a shared type for precinct selections, that supports both single precincts and all precincts:

https://github.com/votingworks/vxsuite/blob/06fc02a33cd27f7a8afee2807dd09bc46a302f32/libs/types/src/precinct_selection.ts#L26

https://github.com/votingworks/vxsuite/blob/06fc02a33cd27f7a8afee2807dd09bc46a302f32/libs/utils/src/precinct_selection.ts#L10-L21

Might be able to use that here

Reply:
Thanks! Didn't know about this type. This file already uses `'all'` as a `PrecinctId` eg [here](https://github.com/votingworks/vxsuite/blob/9aff7f8f6a2fc1fc60d0b0418a9c1b69e6f7f25b/apps/admin/frontend/src/screens/print_test_deck_screen.tsx#L258) and [here](https://github.com/votingworks/vxsuite/blob/9aff7f8f6a2fc1fc60d0b0418a9c1b69e6f7f25b/apps/admin/frontend/src/screens/print_test_deck_screen.tsx#L258). I agree we should refactor to use the `PrecinctSelection` type but might be cleaner to separate to a different PR?

Reply:
Ah yes def feel free to punt to a separate PR!